src/containers/App/App.scss

```diff
@@ -117,3 +117,7 @@ body,
 .g-popup {
     max-width: 500px;
 }
+
+.gc-definition-list__title {
+    @include subheader-2-typography();
+}
```

Comment:
Why do you override this styles? You don't use group titles from `DefinitionList`, you have `<div className={b('list-title')}>{i18n('title_effective-rights')}</div>`

Reply:
For now yes, but it is possible to see groups from Definition list (if some props are not default).
Nonetheless, some fixes for styles are needed!